binding.gyp

```diff
@@ -9,10 +9,11 @@
         "<!@(node -p \"require('node-addon-api').include\")"
       ],
       "cflags!": [ "-fno-exceptions", "-fno-rtti" ],
-      "cflags_cc!": [ "-fno-exceptions", "-fno-rtti" ],
-      "cflags_cc": [ "-std=c++17", "-fexceptions", "-frtti" ],
+      "cflags_cc!": [ "-std=gnu++17", "-fno-exceptions", "-fno-rtti" ],
+      "cflags_cc": [
+        "<!@(pkg-config --cflags parquet arrow)", "-Wall", "-g", "-std=c++17", "-fexceptions", "-frtti" ],
       "cflags": [
-          "<!@(pkg-config --cflags parquet arrow) -Wall -g",
+        "<!@(pkg-config --cflags parquet arrow) -Wall -g",
```

Comment:
Keep the indentation consistent

Reply:
That was to make the indentation consistent between the two actually. I'll go ahead and make it **all** consistent though

Reply:
https://github.com/Comparative-Inc/comparative-parquet/pull/8/commits/5458a4b66c607f5095ee8ee0bf22893944430528